### ISSUE TITLE
Account.UpdatedDateUTC contains local time

### ISSUE THREAD

**Issue by timurka-xalien** [XeroAPI/Xero-NetStandard#121]
Xero API returns Accounts with UpdatedDateUTC containing UTC. However during deserialization SDK converts these values to local time. It is also true for other objects, at least for Organization.

**Comment by jenksguo** [comment]
Current SDK does not integrate timezone information. Fix will be in the next release.

**Comment by jenksguo** [comment]
Hi @timurka-xalien thanks for reporting the issue. 

It has taken some time but we managed to eventually fixed this issue. 

The UpdatedDateUTC now is converted to a DateTime object with DateTime.Kind of Utc: 

![image](https://user-images.githubusercontent.com/41350731/80855015-7a132900-8c80-11ea-84cd-6a9ad01b6ce4.png)

This is now released in [v2.0.1](https://github.com/XeroAPI/Xero-NetStandard/releases/tag/v2.0.1) of the SDK. 

Please note that there is a breaking change between your version and the current SDK. I have changed the namespace of models so you will need to update your inclusion, 

from: 
```csharp
using Xero.NetStandard.OAuth2.Model;
```
to: 
```csharp
using Xero.NetStandard.OAuth2.Model.Accounting;
```
